src/code.cloudfoundry.org/gorouter/mbus/subscriber.go

```diff
@@ -41,7 +41,9 @@ type RegistryMessage struct {
 }
 
 type RegistryMessageOpts struct {
-	LoadBalancingAlgorithm string `json:"loadbalancing"`
+	LoadBalancingAlgorithm string  `json:"loadbalancing"`
+	HashHeader             string  `json:"hash_header"`
```

Comment:
I faintly remember that there was the idea to hash things other than headers, would this then be an additional config option?

Reply:
We are implementing the [RFC in its current state](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0042-hash-based-routing.md#gorouter), which focuses on the Header option.

We are having some discussions about adding more functionality (cookies, URL Params) later on (maybe even before going live with the feature, but not now).